docker-compose.yml

```diff
@@ -3,7 +3,7 @@ version: '2.4'
 services:
 
   nginx:
-    image: "grocy/nginx:v3.0.0-1"
+    image: "bigsisl/nginx:v3.0.0-1"
```

Comment:
Looks like you may have accidentally included a commit that you were testing the build with; could you revert e9d84e5eff8d44e7e08996454fc316b04f819bf5 ?

Reply:
jop, will change it

Reply:
Yep, needed it for testing, I will remove it